pom.xml

```diff
@@ -141,17 +141,18 @@
          upgrade) - START -->
       <!-- Newer versions needed than defined in Boot -->
       <!-- Fixed dependencies -->
-      <rabbitmq.http-client.version>3.5.0.RELEASE</rabbitmq.http-client.version>
+      <rabbitmq.http-client.version>3.12.1</rabbitmq.http-client.version>
```

Comment:
This dependency is not inherited by `spring-boot-dependencies` and could be moved out of the `<!-- Spring boot version overrides (should be reviewed with every boot upgrade) - START -->` section.